lib/List.rb

```diff
@@ -0,0 +1,41 @@
+
+class List
+  DEFAULT_DISHES = {
+                    :burrata => 6, :lasagna => 11,
+                    :risotto => 10, :parmigiana => 12.50,
+                    :quinoa_salad => 7.50
+                    }
+  attr_accessor :available, :food_order
+
+  def initialize
+    @available = Hash.new
+    @food_order = Hash.new { |hash, key| hash[key] = 0 }
```

Comment:
It's a little hard to follow the logic here, they're not very book-ish. These lines should be extracted to a private method.